### ISSUE TITLE
Script Error on Iconlableplus

### ISSUE THREAD

**Issue by shrivatsahosabettu** [tablacus/TablacusExplorerAddons#377]
I am not sure whether this is a issue or additional addon which is missing along with this which need to install. I am getting below Script Error in the addon Iconlabelplus, when used in TablacusExplorer.

![image](https://github.com/tablacus/TablacusExplorerAddons/assets/12061573/201f3111-e0e1-4a08-bf80-c2f4cfec51d6)

```
const Addon_Id = "iconlabelplus";
if (window.Addon == 1) {
	if (WINVER >= 0x600 && await api.IsAppThemed()) {
		AddEvent("Load", function () {
			if (!Addon.ClassicStyle) {
				Sync.IconLabelPlus.SetStyle();
			}
		});
	}
	$.importScript("addons\\" + Addon_Id + "\\sync.js");
}
```

Please suggest and I love this app and thanks for the wonderful work.

**Comment by tablacus** [comment]
I have fixed it on Icon label plus v1.02.
Please give it try.

Regards,

**Comment by shrivatsahosabettu** [comment]
Yes its working thanks for the response.
